### PR TITLE
C3 time axis fix

### PR DESCRIPTION
**Pull request by mlzummo**

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @dtsnell4 @jbrundage - Please Review

File: test/c3chartFactory.js [hunk 1]
```diff
@@ -89,7 +89,22 @@
...
...20140205,843211207580],[20140205,843944253870],[20140206,844159658620],[20140207,844265172090],[20140208,844981442250],[20140210,846364895910],[20140211,847066987280],[20140213,847797417615],[20140214,848094160425],[20140214,848235440245],[20140219,849107136165],[20140224,852474869255],[20140225,854763116670],[20140228,856639460125],[20140228,856773138090],[20140301,857649151595],[20140302,858632952885],[20140303,859444042745],[20140304,860257107255],[20140630,813619339776],[20131111,842380263552],[20140306,860981718395],[20140307,861228144860],[20140308,861356256210],[20140309,862191022525],[20140310,862998410555],[20131108,839430444288],[20131109,842172309888],[20140310,863763402010],[20140312,864499909960],[20140313,865185181035],[20140314,865414689385],[20140314,865533828670],[20140315,866325237000],[20140317,867830391880],[20140317,867103647315],[20131030,836162244864],[20131108,838431302400],[20140630,813520106496],[20131108,840793452288],[20131109,841431800832],[20131108,837709744896],[20140630,797961535872],[20131024,830278809600],[20131107,836810452992],[20131111,844527790080],[20131111,842854300416],[20131108,837618497280],[20131107,837423931392],[20131108,840188184960]])
```
Comment (mlzummo) [inline test/c3chartFactory.js]: remove this testData and go with factory

**Comment by mlzummo** [comment]
@GordonSmith Please Review

File: test/c3chartFactory.js [hunk 2]
```diff
@@ -26,6 +26,18 @@
...
             },
+            timeX: function (callback) {
+                require(["test/DataFactory", "src/chart/Column"], function (DataFactory, Column) {
+                    callback(new Column()
+                        .columns(DataFactory.timeX.default.columns)
+                        .data(DataFactory.timeX.default.data)
+                        
+                        .xAxisType("time")
+                        .xAxisTypeTimePattern("%Y-%m-%dT%H:%M:%S")
+                        .yAxisType("linear")
```
Comment (mlzummo) [inline test/c3chartFactory.js]: this needs to be removed?

**Comment by dtsnell4** [comment]
Looks good to me

**Comment by GordonSmith** [comment]
Should this be targetting 1.6.8?

**Comment by mlzummo** [comment]
i need to remove link to utility class

**Comment by mlzummo** [comment]
removed utility

**Comment by mlzummo** [comment]
@GordonSmith I have a seperate 1.6.8 fix for this should i do a PR?

**Comment by mlzummo** [comment]
http://rawgit.com/mlzummo/Visualization/C3_TIME_FIX/demos/dermatology.html?src/c3chart/Line.timeX

**Comment by mlzummo** [comment]
http://rawgit.com/mlzummo/Visualization/C3_TIME_FIX/demos/dermatology.html?src/c3chart/Column.timeX

**Comment by mlzummo** [comment]
@GordonSmith Fixed so it will display full always if none ...

**Comment by mlzummo** [comment]
closing in favor of 1.6.8 and up merging

**Pull request merged**